server/README.md

```diff
@@ -141,18 +141,33 @@ found here:
 
 
 
-## Adding custom theme
+## Adding a custom theme
 
 To add a custom theme extend the Keycloak image add the theme to the `/opt/jboss/keycloak/themes` directory.
 
 
 
-## Adding custom provider
+## Adding a custom provider
 
 To add a custom provider extend the Keycloak image and add the provider to the `/opt/jboss/keycloak/standalone/deployments/`
 directory.
 
 
+## Adding a custom JGroups discovery protocol
+
+Replacing the default discovery protocols (PING for the UDP stack and MPING for the TCP one) can be achieved by defining
+two additional environment variables:
+
+* `JGROUPS_DISCOVERY_PROTOCOL` - name of the discovery protocol, e.g. DNS_PING
+* `$JGROUPS_DISCOVERY_PROPERTIES` - an optional parameter with the discovery protocol properties in the following format:
+`PROP1=FOO,PROP2=BAR`
+
+Here's an example that adds `DNS_PING` that queries `A` records from DNS with the following query `keycloak.myproject.svc.cluster.local`:
+
+    docker run \
+    -e JGROUPS_DISCOVERY_PROTOCOL=DNS_PING -e \
```

Comment:
Shouldn't the protocol name be `dns.DNS_PING`?

Reply:
Good point! Thanks!